yt/odin/checks/bin/query_tracker_alerts/__main__.py

```diff
@@ -8,7 +8,7 @@ def run_check(secrets, yt_client, logger, options, states):
     logger.info("Checking for relevant query tracker alerts from query tracker stage on cluster %s", cluster_name)
 
     query_tracker_stage_client = YtClient(
```

Comment:
What the difference between `query_tracker_stage_client` and `yt_client` after your change?

Reply:
`query_tracker_stage_client.config` was overridden by new initialization disabling retries.

Reply:
Maybe in such case name it as `yt_client_no_retries` ?

Reply:
@mutexed  – let fix it, and I will merge this PR

Reply:
> @mutexed – let fix it, and I will merge this PR

done